packages/node/package.json

```diff
@@ -39,21 +43,18 @@
     "nock": "^13.1.0",
     "prettier": "^2.3.1",
     "rimraf": "^3.0.2",
-    "supertest": "^6.0.1"
+    "supertest": "^6.0.1",
+    "ts-jest": "^27.0.5",
+    "typescript": "^4.4.2"
   },
   "scripts": {
+    "build": "tsc",
     "lint": "eslint .",
-    "pretest": "npm run lint",
-    "prettier": "prettier --list-different --write \"./**/**.js\"",
+    "prepublish": "npm run build",
```

Comment:
this _should_ work when we publish, but lmk if there's something we need to consider because of lerna or whatever